lib/matplotlib/backend_bases.py

```diff
@@ -1077,10 +1091,17 @@ def set_sketch_params(self, scale=None, length=None, randomness=None):
             The length of the wiggle along the line, in pixels.
         randomness : float, default: 16
             The scale factor by which the length is shrunken or expanded.
+        seed : int, optional
+            Seed for the internal pseudo-random number generator.
         """
+
+        if seed is not None:
+            rcParams['path.sketch_seed'] = seed
+
         self._sketch = (
             None if scale is None
-            else (scale, length or 128., randomness or 16.))
+            else (scale, length or 128., randomness or 16., seed or 0)
```

Comment:
> What do you mean?

Is there a possibility that rcParams['path.sketch'][0] is not none and rcParams['path.sketch'][1] is None? (I do not think so it is possible, but I still wanted to ask you if there was a possibility of that happening because then this will give an error)